backend/substrapp/serializers/ledger/computeplan/serializer.py

```diff
@@ -142,3 +142,9 @@ def get_args(self, data):
     def create(self, validated_data):
         args = self.get_args(validated_data)
         return ledger.create_computeplan(args)
+
+    def update(self, compute_plan_id, validated_data):
+        args = self.get_args(validated_data)
+        del args['tag']
+        args['computePlanID'] = compute_plan_id
```

Comment:
It looks a bit error prone to use the create serialiser to update the asset and handling manually the difference (removing the tag).

It's already quite hard to navigate through the ledger and the view serializers, so that's good for me.